libs/core/src/lib/render-item/render-item.component.ts

```diff
@@ -111,6 +113,8 @@ export class RenderItemComponent extends RenderComponent
   ngOnChanges(changes: SimpleChanges): void {
     if ('item' in changes) {
       this.update();
+    } else if ('context' in changes) {
```

Comment:
What if `item` and `context` are in changes? So the `context` won't be updated

Reply:
No, if `item` changed then all inputs will be updated including `context`.
But if only context was changed then just it's input will be updated.

Reply:
I moved this to more readable method